### PR TITLE
airframe-http-recorder: Delete duplicated server creation methods

### PR DESCRIPTION
**Pull request by takezoe**

### CODE REVIEW

File: airframe-http-recorder/README.md [hunk 1]
```diff
@@ -66,13 +66,15 @@ withResource(HttpRecorder.createReplayOnlyServer(recorderConfig)) { server =>
...
+You can record responses by code instead of recording real interactions with the server.
+
 ```scala
 import wvlet.airframe.http.recorder._
 import wvlet.airframe.control.Control._
 import com.twitter.finagle.http.{Request,Response}
 
 val recorderConfig = HttpRecorderConfig(sessionName="my-recording")
-val response = withResource(HttpRecorder.createProgrammableServer(recorderConfig)) { server =>
+val response = withResource(HttpRecorder.createReplayOnlyServer(recorderConfig)) { server =>
```
Comment (takezoe) [inline airframe-http-recorder/README.md:77]: Using ReplayOnlyServer as ProgrammableServer might be a little weird? InMemoryServer doesn't have variation of creation methods like recordOnly and replayOnly.

Comment (xerial) [reply airframe-http-recorder/README.md:77]: Ok, so how about renaming APIs as createRecordServer and createReplayServer?

**Review by xerial — APPROVED** [review]
Change itself is ok to merge.

File: airframe-http-recorder/README.md (hunk 1 above)
Comment (takezoe) [reply airframe-http-recorder/README.md:77]: Sorry for late reply. That sounds good! I was also thinking that the following namings for consistency:

- createRecordServer
- createServer
- createInMemoryServer

What do you think?

Comment (xerial) [reply airframe-http-recorder/README.md:77]: I'm ok with these changes!

**Pull request merged**